test/tree/OperatorStateRetrieverUnit.tree

```diff
@@ -0,0 +1,37 @@
+.
+└── OperatorStateRetriever tree (*** denotes that integration tests are needed to validate path)
+    ├── when getOperatorState(IRegistryCoordinator,bytes32,uint32) is called
+    │   ├── when the given operator has never registered with the given registryCoordinator (their quorumBitmapHistory has length 0)
+    │   │   └── it should revert with "RegistryCoordinator.getQuorumBitmapIndicesAtBlockNumber: operator has no bitmap history at blockNumber"
+    │   └── when the given operator has registered for the first time with the given registryCoordinator
+    │       ├── after the given blockNumber
+    │       │   └── it should revert with "RegistryCoordinator.getQuorumBitmapIndicesAtBlockNumber: operator has no bitmap history at blockNumber"
+    │       └── before the given blockNumber
+    │           ├── and was not registered for any quorums at the given blockNumber
+    │           │   └── it should return (0, [])
+    │           └── and was registered for a set of quorums at the given blockNumber
+    │               └── it should return their quorum bitmap as the first argument and a list of the list of operators (by id and stake) ordered by index for each quorum in the set ascending by quorumNumber
+    ├── when getOperatorState(IRegistryCoordinator,bytes memory,uint32) is called
+    │   ├── when at any of the given quorumNumbers are not created at the time of call
+    │   │   └── it should revert with "IndexRegistry._operatorCountAtBlockNumber: quorum does not exist"
+    │   ├── when at any of the given quorumNumbers are not created before or at the given blockNumber
+    │   │   └── it should revert with "IndexRegistry._operatorCountAtBlockNumber: quorum did not exist at given block number"
+    │   └── when the given quorumNumbers are created on the given registry registryCoordinator
+    │       └── it should return their quorum bitmap as the first argument and a list of the list of operators (by id and stake) ordered by index for each quorum in the set ascending by quorumNumber
```

Comment:
the 'their' here is super ambiguous. I think you mean `the given operator's`?

Reply:
also maybe `_current_ quorum bitmap`?

Reply:
[3a1b807](https://github.com/Layr-Labs/eigenlayer-middleware/pull/140/commits/3a1b80758dc15258727f0e527415ea94200d00b5)